server/monitor/src/main/resources/templates/org/apache/accumulo/monitor/rest/view/Overview/index.ftl

```diff
@@ -0,0 +1,83 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<html>
+  <head>
+    <title>${title} - Accumulo ${version}</title>
+    <#if refresh gt 0 ><meta http-equiv='refresh' content='${refresh}' /></#if>
+    <meta http-equiv='Content-Type' content='test/html"' />
+    <meta http-equiv='Content-Script-Type' content='text/javascript' />
+    <meta http-equiv='Content-Style-Type' content='text/css' />
+    <link rel='shortcut icon' type='image/jpg' href='/web/favicon.png' />
+    <link rel='stylesheet' type='text/css' href='/web/screen.css' media='screen' />
+    <script src='/web/functions.js' type='text/javascript'></script>
+
+    <!--[if lte IE 8]><script language="javascript" type="text/javascript" src="/web/flot/excanvas.min.js"></script><![endif]-->
+    <script language="javascript" type="text/javascript" src="/web/flot/jquery.js"></script>
+    <script language="javascript" type="text/javascript" src="/web/flot/jquery.flot.js"></script>
+  </head>
+
+  <body>
+    <div id='content-wrapper'>
+      <div id='content'>
+        <div id='header'>
+          <div id='headertitle'>
+            <h1>${title}</h1>
+          </div>
+          <div id='subheader'>Instance&nbsp;Name:&nbsp;${instance_name}&nbsp;&nbsp;&nbsp;Version:&nbsp;${version}
+            <br><span class='smalltext'>Instance&nbsp;ID:&nbsp;${instance_id}</span>
+            <br><span class='smalltext'>${current_date}</span>
+          </div>
+        </div>
+
+        <#include "/templates/sidebar.ftl">
+
+        <div id='main' style='bottom:0'>
+        <#--
```

Comment:
This whole file might be removed/modified when the UI is completed, but thanks for pointing it out!